### ISSUE TITLE
Use without express?

### ISSUE THREAD

**Issue by dfk7677** [nodets/socket.io.users#3]
Hello. I am wondering if this module can be used without express. Can it be used by an socket.io only server?

**Comment by kataras** [comment]
Yes it's working without express, it's a middleware for for socket.io no for express
